README.md

```diff
@@ -9,8 +9,28 @@
 
 This unsigned varint (VARiable INTeger) format is for the use in all the multiformats.
 
-- We have not yet decided on a format yet. When we do, this readme will be updated.
-- We have time. All multiformats are far from requiring this varint.
+There is time to discuss the details of this. All  multiformats are far from requiring this varint.
+
+## Format
+
+Our unsigned varint is an MSB based unsigned varint.
+
+The MSB-based unsigned varint is based on the [varint of the Go standard library](https://golang.org/src/encoding/binary/varint.go), with some modifications:
```

Comment:
It isn't based off Go standard library but off the Protobuf format.

Saying that it is from Go stdlib might be uneasy with some people.